possel/configuration.py

```diff
@@ -0,0 +1,33 @@
+import yaml
+
+
+class Configuration:
+    def __init__(self, default=None):
+        if default:
```

Comment:
When checking for None values we usually do `if value is not None`, in this case there's a commonly used shortcut where we would say `self.config = default or {}`; it makes use of boolean short circuiting.